feat: ASUS ROG page (#149)
Ultramarine-Linux/wiki: src/content/docs/en/anywhere/asus.mdx

---
title: "Asus ROG (Anywhere)"
description: "Asus ROG devices in the Ultramarine Anywhere Initiative"
---

import Alert from "../../../../components/Docs/Alert.astro";

> We have partnered with the Asus Linux project to provide an easy-to-setup Asus Linux setup in Ultramarine.

Reviewer: ```suggestion
We have partnered with the ASUS Linux project to make the ROG experience on Ultramarine easier.
```

# Setup

## Kernel

> 2022 and below Asus ROG models can use the stock Ultramarine kernel.

Reviewer: ```suggestion
ASUS ROG devices from before 2022 can use the stock Ultramarine kernel.
```

> For 2023 and above ROG models, you can use the CachyOS kernel, which may include more drivers for your device then the Fedora kernel that Ultramarine uses.

Reviewer: ```suggestion
ROG devices from 2023 onwards, you should consider using the CachyOS kernel, which may include more drivers for your device than the Fedora kernel used by default.
```

Ultramarine includes a `umcli` tweak to easily enable or disable the CachyOS kernel, information can be found [here](../usage/umcli)

## Packages

> We have the following Asus Linux/Asus ROG device support packages in Terra:

Reviewer: ```suggestion
We have the following ASUS Linux/ASUS ROG support packages in Terra:
```

- [asusctl and rog-control-center](https://gitlab.com/asus-linux/asusctl)
- [supergfxctl](https://gitlab.com/asus-linux/supergfxctl)
- [gnome-shell-extension-gpu-switcher-supergfxctl](https://extensions.gnome.org/extension/7018/gpu-supergfxctl-switch/)

## TuneD Issues
If you are having issues with `tuned`, you may want to instead use `power-profiles-daemon`:

> 

Reviewer: maybe link out to asus linux on how to tell if tuned is the issue

```
sudo dnf install power-profiles-daemon --allowerasing
systemctl enable --now power-profiles-daemon.service
```

### Improve Battery on AMD Laptops

> If you are using `power-profiles-daemon`, it has two functions that are disabled by default: Panel power savings and AMDGPU Dynamic power management. These are dedicated to AMD iGPU, and you must follow these steps:

Reviewer: ```suggestion
If you are using `power-profiles-daemon`, two features useful for ROG laptops are disabled by default; Panel power savings and AMDGPU Dynamic power management. These are specific to AMD iGPUs. Follow these steps to enable them:
```

---
title: "ASUS ROG (Anywhere)"
description: "ASUS ROG devices in the Ultramarine Anywhere Initiative"
---

import Alert from "../../../../components/Docs/Alert.astro";

We have partnered with the [ASUS Linux project](https://asus-linux.org/) to make the ASUS ROG experience on Ultramarine very user-friendly.

## Setup

### Kernel

ASUS ROG devices from or before 2022 can use the stock Ultramarine kernel.
If you have a ROG devices from 2023 onwards, you should consider using the CachyOS kernel, as it includes more drivers for your device than the Fedora kernel used by default.
Ultramarine includes a `umcli` tweak to easily enable or disable the CachyOS kernel, information can be found [here](../usage/umcli)

<Alert type="info">

>   Fyra Labs/Ultramarine and ASUS Linux are both a part of the Open Gaming Collective and are working on centralizing all needed ROG patches into an Open Gaming Collective kernel.

Reviewer: ```suggestion
  Fyra Labs and ASUS Linux are both members of the Open Gaming Collective. We're working on centralizing all needed ROG patches into an OGC kernel.
```

  Stay tuned for more updates!
</Alert>

### Packages

We have the following ASUS Linux/ASUS ROG support packages in Terra:
- [asusctl and rog-control-center](https://gitlab.com/asus-linux/asusctl)

### TuneD Issues
If you are having issues with `tuned`, you may want to instead use `power-profiles-daemon`:

```
sudo dnf install power-profiles-daemon --allowerasing

> systemctl enable --now power-profiles-daemon.service

Reviewer: they may need to disable tuned-ppd that we enable oob, check with asus linux

```

#### Improve Battery on AMD Laptops

If you are using `power-profiles-daemon`, two features useful for ROG laptops are disabled by default; Panel power savings and AMDGPU Dynamic power management. These are specific to AMD iGPUs. Follow these steps to enable them:

Enable panel power savings:

```
powerprofilesctl configure-action amdgpu_panel_power --enable
```

Make sure this is above 0:

```
cat /sys/class/drm/card2/card2-eDP-2/amdgpu/panel_power_savings
```

Enable AMDGPU Dynamic power management:

```
powerprofilesctl configure-action amdgpu_dpm --enable
```

For more information: [PPD Documentation](https://gitlab.gnome.org/Infrastructure/Mirrors/lorry-mirrors/gitlab_freedesktop_org/hadess/power-profiles-daemon)

## Notes

- When installing `asusctl`, these systemd services will be pre-enabled: `asusd.service`, `asus-shutdown.service`.

> - ROG Control Center is available in your app store, just search "ROG"!

Reviewer: ```suggestion
- ROG Control Center is available in your app store!
```
